exo-add/features/exoservice-go.feature

```diff
@@ -0,0 +1,79 @@
+Feature: scaffolding an ExoService written in ES6
+
+  As a JavaScript developer adding features to an Exosphere application
+  I want to have an easy way to scaffold an empty ExoService
+  So that I don't waste time copy-and-pasting a bunch of code.
+
+  - run "exo add service <name> exoservice-es6"
+    to add a new exo-service written in ES6 to the current application
+  - run "exo add service" to add the service interactively
+
+
+  Scenario: calling with all command line arguments
+    Given I am in the root directory of an empty application called "test app"
+    When running "exo-add service --service-role=users --service-type=users --author=test-author --template-name=exoservice-go --model=user --description=testing" in this application's directory
+    Then my application contains the file "application.yml" with the content:
+      """
+      name: test app
+      description: Empty test application
+      version: 1.0.0
+
+      bus:
+        type: exocom
+        version: 0.21.7
```

Comment:
Current exocom version is `0.21.8`. @kevgo should we be updating this in the tests with each exocom release?